### PR TITLE
Added course on cloud computing

### PR DESCRIPTION
**Pull request by metabiswadeep**
## What does this PR do?
Added a free course on cloud computing.

### Description
Added a free course on cloud computing to master google cloud and also enable you to take the google cloud certification exam and ace it.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -267,6 +267,7 @@
 
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
 * [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
+* [Google Cloud Certification Course](https://www.youtube.com/watch?v=UGRDM86MBIQ) - Free Code Camp
```
Comment (eshellman) [inline courses/free-courses-en.md:270]: For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Comment (metabiswadeep) [reply courses/free-courses-en.md:270]: This is a 6hr video for the complete course theory + code along. So can be considered as a playlist in itself. Name of producer and creator will be added after you review this comment. @eshellman

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -267,6 +267,7 @@
 
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
 * [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
+* [Google Cloud Certification Course](https://www.youtube.com/watch?v=UGRDM86MBIQ) - Andrew Brown - Free Code Camp
```
Comment (davorpa) [inline courses/free-courses-en.md:270]: Don't invent titles. Use the one provided by original source.

```suggestion
* [Google Cloud Digital Leader Certification Course - Pass the Exam!](https://www.youtube.com/watch?v=UGRDM86MBIQ) - Andrew Brown (freeCodeCamp)
```

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md]: It's ok for me too. 6h video but without chapters integrated in timeline

**Comment by metabiswadeep** [comment]
Please check now @davorpa

**Comment by eshellman** [comment]
please resolve conflicts.

**Pull request closed without merging**